pom.xml

```diff
@@ -64,6 +64,7 @@
         <dep.docker-java.version>3.2.8</dep.docker-java.version>
         <dep.coral.version>1.0.60</dep.coral.version>
         <dep.confluent.version>5.5.2</dep.confluent.version>
+        <dep.hudi.version>0.7.0</dep.hudi.version>
```

Comment:
Could we upgrade to 0.8.0?

Reply:
I test the code based on Hudi 0.8.0 before, but there is an error showing that one API  does not support. I will try it again.